### ISSUE TITLE
Placeholder sometimes starts at incorrect position in horizontal lists

### ISSUE THREAD

**Issue by adambuczynski** [marceljuenemann/angular-drag-and-drop-lists#293]
See behaviour:

![ss](https://www.dropbox.com/s/jwes7i3hqmvjovt/placeholder%20wrong.gif?dl=1)

Code:

```html
    <ul class="sort-container" dnd-list="suggestions">
      <li ng-repeat="suggestion in suggestions" dnd-draggable="suggestion" dnd-moved="suggestions.splice($index, 1)" dnd-horizontal-list="true">
        {{suggestion.name}}
      </li>
    </ul>
```

As you can see, the placeholder is sometimes initialised over the right element instead of the one directly below it. Any idea what might be causing this?

Note that the items are displayed using CSS flexbox.

**Comment by adambuczynski** [comment]
Same happening with items floating left.

**Comment by mightyplow** [comment]
I have the same problem. Did you find any solution for that?

**Comment by mightyplow** [comment]
But I think it's worth to mention, that it seems, like you are using the dnd-horizontal-list attribute on the wrong elements since it's actually an attribute to dnd-list and you use it on dnd-draggable, or do I get something wrong?

**Comment by adambuczynski** [comment]
No, I didn't. I moved to use [SortableJS](http://rubaxa.github.io/Sortable/) instead, which I found to be more elegant and easier to use with fewer issues.

**Comment by marceljuenemann** [comment]
Indeed, your code in the OP has the dnd-horizontal-list on the wrong element. I also implemented a new positioning algorithm in the dev branch a while ago, might be fixed in 2.0 then.
